wagtailnhsukfrontend/blocks.py

```diff
@@ -49,3 +50,34 @@ class InsetTextBlock(RichTextBlock):
 
     class Meta:
         template = 'wagtailnhsukfrontend/inset_text.html'
+
+
+class DetailsBlock(StructBlock):
+
+    details_title = CharBlock(label="Title", required=True)
+    details_body = RichTextBlock(label="Body", required=True)
+
+    class Meta:
+        template = 'wagtailnhsukfrontend/details.html'
+
+
+class ExpanderBlock(StructBlock):
+
+    def get_context(self, value, parent_context=None):
+        context = super().get_context(value, parent_context)
+        context['expander'] = ' nhsuk-expander'
+        return context
+
+    details_title = CharBlock(label="Title", required=True)
+    details_body = RichTextBlock(label="Body", required=True)
+
+    class Meta:
+        template = 'wagtailnhsukfrontend/details.html'
+
+
+class GroupExpanderBlock(StructBlock):
```

Comment:
Lets call this `ExpanderGroupBlock` instead of `GroupExpanderBlock` to keep consistent with the frontend library naming

Reply:
Done.